studybuddies/src/app/components/student/create-class-student/create-class-student.component.ts

```diff
@@ -7,9 +10,42 @@ import { Component, OnInit } from '@angular/core';
 })
 export class CreateClassStudentComponent implements OnInit {
 
-  constructor() { }
+  constructor(public roomService: RoomService) { }
 
   ngOnInit(): void {
   }
 
+  createRoom(form: NgForm){
+
+    let date = form.value.date.split("-");
+    let iDay = form.value.iTime.split(":");
+    let fDay = form.value.fTime.split(":");
+
+    let room: Room = {
+      id_user : "",
+      description : form.value.description,
+      university : form.value.university,
+      degree : form.value.degree,
+      subject : form.value.subject,
+      starting_time : new Date(parseInt(date[0]), parseInt(date[1])-1, parseInt(date[2]),
+      parseInt(iDay[0]), parseInt(iDay[1])),
+      ending_time : new Date(parseInt(date[0]), parseInt(date[1])-1, parseInt(date[2]),
+      parseInt(fDay[0]), parseInt(fDay[1])),
+      price_per_hour : 0,
+      is_private : false,
+      date : new Date(),
+      iTime : new Date(),
+      fTime : new Date()
+    };
+
+    console.log(room);
+
+    this.roomService.createRoom(room).subscribe(
+      res => {
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7199829)
```suggestion
      (res) => {
```